src/deployment/deploy.py

```diff
@@ -694,42 +689,50 @@ def set_instance_config(self) -> None:
             tenants.append(tenant)
         update_allowed_aad_tenants(config_client, tenants)
 
-    def create_eventgrid(self) -> None:
-        logger.info("creating eventgrid subscription")
-        src_resource_id = self.results["deploy"]["fuzz-storage"]["value"]
-        dst_resource_id = self.results["deploy"]["func-storage"]["value"]
-
+    def remove_eventgrid(self) -> None:
```

Comment:
old one has to be removed, can't be re-used ?

Reply:
no we can't reuse it because it was associated with the auto generated topic that we want to remove